teuthology/describe_tests.py

```diff
@@ -243,6 +243,8 @@ def tree_with_info(cur_dir, fields, include_facet, prefix, rows,
     has_yamls = any([x.endswith('.yaml') for x in files])
     facet = os.path.basename(cur_dir) if has_yamls else ''
     for i, f in enumerate(files):
+        if f == ".qa":
```

Comment:
This is not a loop detection code, though it makes more sense to use `f.startswith('.')` to ignore "hidden" files, which is supposed to be a case if I understand correctly the idea of having '.qa' links.

Reply:
Good idea. Done.